src/help.c

```diff
@@ -188,6 +188,7 @@ static void help_draw_global(ToxWindow *self)
     wprintw(win, "  /log <on> or <off>         : Enable/disable logging\n");
     wprintw(win, "  /myid                      : Print your Tox ID\n");
     wprintw(win, "  /group <name>              : Create a new group chat\n");
+    wprintw(win, "  /join <chatid>             : Join a groupchat using a Chat ID\n");
```

Comment:
You need to adjust the size of the help menu to account for the new entries in the `help_onkey()` function.

Reply:
I've adjusted the height for both sections of the menu. I cannot see the difference on my side though, does it look correct to you?